docs/install/upgrade.md

```diff
@@ -7,7 +7,8 @@ This article walks you through how to upgrade your Coder server.
 > Coder does not support rollbacks.
 
 To upgrade your Coder server, simply reinstall Coder using your original method
-of [install](../install).
+of [install](../install). If you are using the Official Coder AMI on AWS, use the
+first option to upgrade.
```

Comment:
thanks @d1str0 !

I don't currently have access to an AMI that needs an update. Can you tell me if the first option says "Upgrade" or if it's just the first in a list? I want to be able to include the exact text someone will encounter, just in case the AWS UI changes the order or adds an option.

Reply:
There is no way to upgrade from within the AMI. My intention here was that those who are using the AMI should use the "Via install.sh" option. As in the first option on this documentation page. Does that make sense? I wasn't sure how best to word it.

Reply:
"To upgrade Coder server if you are using the AWS AMI, simply run `curl -L https://coder.com/install.sh | sh` on the machine."

Reply:
that totally makes sense

I have an idea that might help - I just pointed this to a new branch and I'll be right back with a PR

Reply:
continued in https://github.com/coder/coder/pull/16973